apps/E2E/src/TabList/specs/TabList.spec.win.ts

```diff
@@ -0,0 +1,190 @@
+import { Attribute, AttributeValue, Keys, TAB_A11Y_ROLE, TABITEM_A11Y_ROLE } from '../../index.consts';
+import {
+  FIRST_TAB_ACCESSIBILITY_LABEL,
+  FIRST_TAB_KEY,
+  FOURTH_TAB_KEY,
+  SECOND_TAB_KEY,
+  SECOND_TAB_LABEL,
+  TABLIST_ACCESSIBILITY_LABEL,
+} from '../consts';
+import TabListPageObject from '../pages/TabListPageObject';
+
+describe('TabList Testing Initialization', () => {
+  it('Wait for app load', async () => {
+    await TabListPageObject.waitForInitialPageToDisplay();
+    expect(await TabListPageObject.isInitialPageDisplayed())
+      .withContext(TabListPageObject.ERRORMESSAGE_APPLOAD)
+      .toBeTruthy();
+  });
+
+  it('Click and navigate to TabList test page', async () => {
+    await TabListPageObject.navigateToPageAndLoadTests(true);
+    expect(await TabListPageObject.isPageLoaded())
+      .withContext(TabListPageObject.ERRORMESSAGE_PAGELOAD)
+      .toBeTruthy();
+
+    expect(await TabListPageObject.didAssertPopup())
+      .withContext(TabListPageObject.ERRORMESSAGE_ASSERT)
```

Comment:
What is `.withContext()` for? Should we make a task to normalize this pattern?

Reply:
Before, we were passing error messages in the `.toBeTruthy` / `.toBeFalsy` methods within specs. This is deprecated in Jasmine - `withContext` is the correct method to pass the error message if the test fails.

Reply:
It would probably be a good idea to normalize this across e2e tests.

Reply:
Awesome! Room to improve - do you mind creating a task for this? Good error messages are crucial :)